packages/rich-text/src/plugins/Table/normalizers.ts

```diff
@@ -0,0 +1,74 @@
+import { PlateEditor } from '@udecode/plate';
+import { Element, Node, Transforms } from 'slate';
+import { BLOCKS, CONTAINERS } from '@contentful/rich-text-types';
+
+import { CustomElement } from '../../types';
+import { replaceNode, getTextAt } from '../../helpers/editor';
+import { Normalizer, withNormalizer } from '../../helpers/normalizers';
+
+const isTable = (node: CustomElement) => {
+  return node.type === BLOCKS.TABLE;
+};
+
+const isTableCell = (node: CustomElement) => {
+  return node.type === BLOCKS.TABLE_CELL || node.type === BLOCKS.TABLE_HEADER_CELL;
+};
+
+/**
+ * Normalizes TABLE_CELL & TABLE_HEADER_CELL nodes
```

Comment:
👍

Reply:
Are we forgetting to normalize table rows?

Reply:
That's a different ticket